lighthouse-core/audits/cache-start-url.js

```diff
@@ -37,21 +37,18 @@ class CacheStartUrl extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    let cacheHasStartUrl = false;
-    const manifest = artifacts.Manifest.value;
-    const cacheContents = artifacts.CacheContents;
-
-    if (!(manifest && manifest.start_url && manifest.start_url.value)) {
+    if (!artifacts.Manifest || !artifacts.Manifest.value) {
```

Comment:
well i guess i'm saying we should handle the invalid JSON case via `response.errors` in the gatherer, so we can be more clear here.  otherwise these are the only audits that have to differentiate between two error states (plus they can't say which they're in)